### PR TITLE
Development

### PR DESCRIPTION
**Pull request by IBTechRaj**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<html lang="en">
```
Comment (zlayabekrija) [inline index.html]: this is extra html tag please remove it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			<p>including Mint.<a href="">Learn more</a> </p>
+		</div>
+		<form action="">
+			<div>
+				<label for="mail" id="mail-label">Email address</label>
+				<input type="email" id="mail" name="user_email" />
+			</div>
+			<div>
+				<label for="phone" id="phone-label">Phone (recommended)</label>
+				<input type="phone" id="phone-1"  name="user_phone" />
```
Comment (zlayabekrija) [inline index.html]: input type should be tel

File: index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		</div>
+	</div>
+	<div id="login-form"> 
+		<div id="form-text">
+			<h2 id="first">One Account.</h2>
+			<h2 id="second">Everything Intuit.</h2>
+			<p>Create an account for all of Intuit's products </p>
+			<p>including Mint.<a href="">Learn more</a> </p>
+		</div>
+		<form action="">
```
Comment (zlayabekrija) [inline index.html]: should `action="/"`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+				<p></p>
+			</div>
+			<div>
+				<label for="password" id="password-label">Password</label>
+				<input type="password" id="password" name="user_password" />
+			</div>
+			<br>
+
+			<div>
+				<button type="submit" id="create-button"> Create Account </button>
```
Comment (zlayabekrija) [inline index.html:52]: the button should be centered

File: style.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+}
+body{
+	background-color: #ECEEF1;
+	font-family: AvenirNext,Avenir,"Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-weight: 400;
+}
+#header{
+background-color: #ECEEF1;
+height: 100px;
+margin-left: 40%;
```
Comment (zlayabekrija) [inline style.css]: these are huge margins, please find different approach.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+	margin-left: 5px;
+	width: 100px;
+	height: 35px;
+	float: right;
+	background-color: #0077C5;
+	color: white;
+	border-radius: 7px;
+}
+#top-bar p {
+	margin-right: 140px;
```
Comment (zlayabekrija) [inline style.css:30]: this line is unnecessary if margin 0 !important is used

File: style.css [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+	padding-bottom: 5px;
+	margin: 0 !important;
+}
+#top-bar p a {
+	text-decoration: none;
+	color: blue;
+	margin-top: 5px;
+}
+form{
+	margin-top: 5%;
```
Comment (zlayabekrija) [inline style.css]: same here 5% margin is too big

File: style.css [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+}
+#form-text a{
+	text-decoration: none;
+	color: blue;
+}
+#form-text h2{
+	margin: 0 !important;
+}
+#form-text #first{
+	padding-top: 5%;
```
Comment (zlayabekrija) [inline style.css]: please use px or rem or em for both padding and margin

File: style.css [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+border: solid 1px #C6CFCF;
+
+}
+#form-text{
+	text-align: center;
+}
+
+
+#bottom-text{
+	margin: 5% 5%;
```
Comment (zlayabekrija) [inline style.css]: change this please

File: style.css [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+}
+#footer{
+background-color: #ECEEF1;
+height: 50px;
+margin-top: 20px;
+}
+
+
+#intuit{
+	margin-left: 30%;
```
Comment (zlayabekrija) [inline style.css]: margin too big, please change it

File: style.css [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+	color: white;
+	border-radius: 5px;
+
+
+}
+#phone-label-1{
+	font-size: 12px;
+}
+#mail-label, #phone-label, #phone-label-1, #password-label {
+	margin-left: 7%;
```
Comment (zlayabekrija) [inline style.css]: same do pixels

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
It was great try, but there are some changes to be made.
Please follow the instructions.
After completing, submit new review request and happy coding.

**Comment by DanielMitiku** [comment]
Hey Zlayabekrija, can you please see our correction.

**Review by zlayabekrija — APPROVED** [review]
Great work, thumbs up 👍 from me.

**Pull request merged**